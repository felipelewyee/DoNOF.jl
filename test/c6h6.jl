using DoNOF

mol = """
0 1
  C    1.2116068   -0.6995215    0.0000000
  C    1.2116068    0.6995215    0.0000000
  C    0.0000000    1.3990430    0.0000000
  C   -1.2116068    0.6995215   -0.0000000
  C   -1.2116068   -0.6995215   -0.0000000
  C   -0.0000000   -1.3990430   -0.0000000
  H    2.1489399   -1.2406910    0.0000000
  H    2.1489399    1.2406910    0.0000000
  H    0.0000000    2.4813820    0.0000000
  H   -2.1489399    1.2406910   -0.0000000
  H   -2.1489399   -1.2406910   -0.0000000
  H   -0.0000000   -2.4813820   -0.0000000
"""

bset,p = DoNOF.molecule(mol,"cc-pvdz",spherical=true)

p.ipnof = 8

DoNOF.energy(bset,p)

