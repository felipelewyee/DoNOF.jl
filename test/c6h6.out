Computing Integrals
Elapsed time:    3.11 Seconds

Number of basis functions                   (NBF)    = 114
Number of auxiliary basis functions         (NBFAUX) = 558
Inactive Doubly occupied orbitals up to     (NO1)    = 0
No. considered Strongly Doubly occupied MOs (NDOC)   = 21
No. considered Strongly Singly occupied MOs (NSOC)   = 0
NO. of Weakly occ. per St. Doubly occ.  MOs (NCWO)   = 4
Dimension of the Nat. Orb. subspace         (NBF5)   = 105
No. of electrons                                     = 42
    No. of alpha electrons                           = 21
    No. of beta electrons                            = 21
Multiplicity                                         = 1

Geometry
========
 C       1.2116068      -0.6995215       0.0000000
 C       1.2116068       0.6995215       0.0000000
 C       0.0000000       1.3990430       0.0000000
 C      -1.2116068       0.6995215      -0.0000000
 C      -1.2116068      -0.6995215      -0.0000000
 C      -0.0000000      -1.3990430      -0.0000000
 H       2.1489399      -1.2406910       0.0000000
 H       2.1489399       1.2406910       0.0000000
 H       0.0000000       2.4813820       0.0000000
 H      -2.1489399       1.2406910      -0.0000000
 H      -2.1489399      -1.2406910      -0.0000000
 H      -0.0000000      -2.4813820      -0.0000000

No violations of the orthonormality
No violations of the orthonormality
 
PNOF8/ADABelief Calculation
==================
 
  Nitext  Nit_orb  Time   Nit_occ  Time         Eelec          Etot           Ediff      maxdiff    MaxGrad_orb  Grad_occ
     1      10    1.0e+01    6    1.2e+01  -427.37900892  -224.36009242 -10426.37900892   0.242743   9.7e-01   1.8e-04 0.00
     2      10    1.5e+00   35    3.0e-01  -432.74112526  -229.72220875    -5.36211633   0.120098   4.8e-01   2.1e-04 0.02
     3      10    1.4e+00   40    2.9e-01  -434.36359688  -231.34468037    -1.62247162   0.111772   4.5e-01   2.6e-04 0.03
     4      10    1.6e+00   25    2.2e-01  -434.47981717  -231.46090067    -0.11622029   0.037067   1.5e-01   2.3e-04 0.03
     5      10    1.4e+00   18    2.1e-01  -434.50880584  -231.48988933    -0.02898867   0.019559   7.8e-02   1.9e-04 0.03
     6      10    1.5e+00   16    1.7e-01  -434.52635922  -231.50744271    -0.01755338   0.019420   7.8e-02   1.4e-04 0.03
     7      10    1.4e+00   12    1.3e-01  -434.53648863  -231.51757213    -0.01012942   0.017561   7.0e-02   2.1e-04 0.03
     8      10    1.4e+00   12    1.0e-01  -434.54154924  -231.52263274    -0.00506061   0.016877   6.8e-02   1.9e-04 0.03
     9      10    1.4e+00   11    1.3e-01  -434.54510513  -231.52618863    -0.00355589   0.013947   5.6e-02   1.8e-04 0.03
    10      10    1.6e+00   11    1.1e-01  -434.54799974  -231.52908323    -0.00289460   0.014829   5.9e-02   1.8e-04 0.03
    11      10    1.5e+00   14    1.5e-01  -434.55044741  -231.53153091    -0.00244768   0.014171   5.7e-02   1.5e-04 0.03
    12      10    1.6e+00   13    1.9e-01  -434.55445148  -231.53553497    -0.00400406   0.015777   6.3e-02   1.5e-04 0.03
    13      10    1.6e+00   10    1.0e-01  -434.55793723  -231.53902073    -0.00348576   0.013879   5.6e-02   2.3e-04 0.03
    14      10    1.6e+00   10    1.2e-01  -434.55990949  -231.54099298    -0.00197225   0.014720   5.9e-02   1.8e-04 0.03
    15      10    1.5e+00   10    9.6e-02  -434.56139471  -231.54247821    -0.00148523   0.014214   5.7e-02   1.6e-04 0.03
    16      10    1.4e+00   11    1.0e-01  -434.56206751  -231.54315101    -0.00067280   0.012919   5.2e-02   1.1e-04 0.03
    17      10    1.3e+00    8    1.0e-01  -434.56264798  -231.54373147    -0.00058047   0.013864   5.5e-02   2.1e-04 0.03
    18      10    1.5e+00    8    1.1e-01  -434.56315062  -231.54423412    -0.00050264   0.015281   6.1e-02   1.8e-04 0.03
    19      10    1.4e+00    8    1.2e-01  -434.56398653  -231.54507003    -0.00083591   0.016871   6.7e-02   2.3e-04 0.03
    20      10    1.4e+00   12    1.5e-01  -434.56560107  -231.54668457    -0.00161454   0.014962   6.0e-02   1.2e-04 0.03
    21      10    1.4e+00    8    9.4e-02  -434.56747686  -231.54856036    -0.00187578   0.014522   5.8e-02   1.6e-04 0.03
    22      10    1.5e+00    7    1.0e-01  -434.56836424  -231.54944773    -0.00088738   0.013647   5.5e-02   1.4e-04 0.03
    23      10    1.5e+00    6    9.4e-02  -434.56895997  -231.55004346    -0.00059573   0.014986   6.0e-02   1.5e-04 0.03
    24      10    1.6e+00    0    5.1e-02  -434.56901530  -231.55009880    -0.00005533   0.014986   6.0e-02   1.5e-04 0.03
    25      16    2.1e+00    4    4.9e-02  -434.57815800  -231.55924149    -0.00914270   0.000432   1.7e-03   1.8e-04 0.03
 
RESULTS OF THE OCCUPATION OPTIMIZATION
========================================
   1    1.9999446  -11.23526441
   2    1.9999196  -11.23518692
   3    1.9998893  -11.23473257
   4    1.9999072  -11.23443298
   5    1.9999368  -11.23435662
   6    1.9999342  -11.23407287
   7    1.9772479  -0.85940060
   8    1.9768839  -0.84447399
   9    1.9766682  -0.84097769
  10    1.9765777  -0.83495865
  11    1.9767742  -0.83422509
  12    1.9756027  -0.81158792
  13    1.9740607  -0.69466209
  14    1.9737748  -0.69115145
  15    1.9722361  -0.68404961
  16    1.9731326  -0.68376156
  17    1.9720178  -0.68333097
  18    1.9724399  -0.68247779
  19    1.9658484  -0.41722459
  20    1.9659546  -0.41384225
  21    1.9665513  -0.40784163
  22    0.0313736  -0.03523523
  23    0.0284910  -0.03287015
  24    0.0266341  -0.03104164
  25    0.0177956  -0.02120512
  26    0.0175532  -0.02103002
  27    0.0176865  -0.02120182
  28    0.0179822  -0.02133363
  29    0.0189429  -0.02199768
  30    0.0188410  -0.02200267
  31    0.0180851  -0.02722933
  32    0.0183794  -0.02760096
  33    0.0187222  -0.02783437
  34    0.0177245  -0.02706876
  35    0.0182719  -0.02751751
  36    0.0184421  -0.02765687
  37    0.0000212  -0.00023465
  38    0.0000247  -0.00027489
  39    0.0000274  -0.00032097
  40    0.0000316  -0.00036204
  41    0.0000254  -0.00028283
  42    0.0000270  -0.00030008
  43    0.0008211  -0.00151059
  44    0.0044515  -0.00620800
  45    0.0044294  -0.00623407
  46    0.0046124  -0.00675267
  47    0.0046294  -0.00664629
  48    0.0049041  -0.00723691
  49    0.0043948  -0.00639447
  50    0.0036952  -0.00536554
  51    0.0038598  -0.00548725
  52    0.0030556  -0.00573887
  53    0.0038033  -0.00710280
  54    0.0032117  -0.00580583
  55    0.0029271  -0.00580238
  56    0.0026784  -0.00478180
  57    0.0025732  -0.00470814
  58    0.0000199  -0.00022715
  59    0.0000202  -0.00022885
  60    0.0000255  -0.00028428
  61    0.0000296  -0.00033172
  62    0.0000243  -0.00027145
  63    0.0000191  -0.00022255
  64    0.0006379  -0.00125344
  65    0.0006148  -0.00122326
  66    0.0025671  -0.00359718
  67    0.0034247  -0.00507711
  68    0.0034674  -0.00508607
  69    0.0034891  -0.00512762
  70    0.0034664  -0.00508943
  71    0.0025277  -0.00409735
  72    0.0024599  -0.00411539
  73    0.0026222  -0.00466858
  74    0.0005641  -0.00129790
  75    0.0008752  -0.00194790
  76    0.0021627  -0.00392467
  77    0.0012283  -0.00253521
  78    0.0010354  -0.00219884
  79    0.0000193  -0.00021833
  80    0.0000107  -0.00011048
  81    0.0000211  -0.00023416
  82    0.0000256  -0.00029258
  83    0.0000198  -0.00022502
  84    0.0000049  -0.00004261
  85    0.0006161  -0.00122043
  86    0.0004881  -0.00092155
  87    0.0005211  -0.00098635
  88    0.0017274  -0.00294959
  89    0.0023321  -0.00385635
  90    0.0007877  -0.00151119
  91    0.0019205  -0.00322537
  92    0.0010593  -0.00194096
  93    0.0007787  -0.00147049
  94    0.0006343  -0.00145584
  95    0.0004789  -0.00117344
  96    0.0006132  -0.00143853
  97    0.0005174  -0.00125535
  98    0.0009375  -0.00208895
  99    0.0007014  -0.00160498
 100    0.0000054  -0.00004509
 101    0.0000076  -0.00007327
 102    0.0000188  -0.00021397
 103    0.0000240  -0.00027247
 104    0.0000111  -0.00011314
 105    0.0000044  -0.00003974
 
----------------
 Final Energies 
----------------
Final NOF Total Energy =    -231.5592415
 
Total Orbital Internal Iterations = 256
Total Occupation Internal Iterations = 315
 
Elapsed time:   72.48 Seconds

---------------------------------
   Lowdin Population Analysis    
---------------------------------
 Idx  Atom   Population   Charge 
---------------------------------
  1     C     6.153      -0.153
  2     C     6.150      -0.150
  3     C     6.151      -0.151
  4     C     6.149      -0.149
  5     C     6.152      -0.152
  6     C     6.148      -0.148
  7     H     0.849       0.151
  8     H     0.849       0.151
  9     H     0.849       0.151
 10     H     0.849       0.151
 11     H     0.850       0.150
 12     H     0.850       0.150

---------------------------------
   M Diagnostic: 0.03
---------------------------------

